src/ClientInfo.h

```diff
@@ -74,7 +74,7 @@ class ClientInfo : public hash_link
     bool hasQueue() const;  ///< whether any clients are waiting for write quota
     bool hasQueue(const CommQuotaQueue*) const;  ///< has a given queue
     unsigned int quotaEnqueue(int fd); ///< client starts waiting in queue; create the queue if necessary
-    int quotaPeekFd() const; ///< retuns the next fd reservation
+    int quotaPeekFd() const; ///< returns the next fd reservation
```

Comment:
What rule is broken here?
We can solve this by adding "retuns" to our white list.

Reply:
Amos meant that `returns` should be `\returns`. This problem is not related to this PR (and, IMO, should not be fixed in this PR).

In this change request, Amos is documenting a case where this PR modifies a line but does not fix all the problems on that line. For regular PRs, this would normally mean that the author should fix all those other problems as well. This is _not_ a regular PR. The "author" here is, essentially, a script. That is why Amos correctly said that this problem "will need a separate manual fix".

I suggest interpreting the word "separate" as "not in this PR" :-).

<!--- Implementing that manual fix here would be a minor problem, but it will destroy the clarity of this PR scope and, technically, open this PR to hundreds of similar requests (which, in aggregate, would become a major issue, potentially derailing the whole PR). I am optimistic that we can avoid that sad fate for this PR. -->

Reply:
So the change from "retuns" to "returns" is not in itself an error?
Then no need to add it to the white list.

Reply:
> So the change from "retuns" to "returns" is not in itself an error?
>  Then no need to add it to the white list.

Correct on both counts.